### PR TITLE
Datafeeder api proposal

### PR DESCRIPTION
**Pull request by groldan**

### CODE REVIEW

File: datafeeder/api.yaml [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+        401:
+          description: 'Not authenticated'
+        403:
+          description: 'Forbidden. User has no prviledges to query other users jobs'
+
+  /upload/all:
+    get:
+      tags:
+        - File Upload
+      description: 'Poll the status of a given upload job by id'
```
Comment (fgravin) [inline datafeeder/api.yaml:59]: `of a given upload job by id` confusing with `all`

File: datafeeder/api.yaml [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+        401:
+          description: 'Not authenticated'
+        403:
+          description: 'Forbidden. User has no prviledges to upload files'
+  
+  /upload/{jobId}:
+    get:
+      tags:
+        - File Upload
+      description: 'Poll the status of all the upload jobs. Access restricted to administrators'
```
Comment (fgravin) [inline datafeeder/api.yaml:73]: switch description with `upload/all`

**Review by fgravin — COMMENTED** [review]
great, small switch between 2 descriptions

File: datafeeder/api.yaml (hunk 1 above)
Comment (groldan) [reply datafeeder/api.yaml]: Thanks, fixed at 4fc1613

File: datafeeder/api.yaml (hunk 2 above)
Comment (groldan) [reply datafeeder/api.yaml]: Thanks, fixed at 4fc1613

**Comment by landryb** [comment]
`api.yaml` has several instances of `User has no prviledges access the requested job` which doesnt feel 'right', probably `User has no priviledges to access the requested job`

**Pull request merged**